APIBlueprint.tmLanguage

```diff
@@ -437,6 +453,28 @@
     </dict>
 
     <dict>
+      <key>captures</key>
+      <dict>
+        <key>2</key>
+        <dict>
+          <key>name</key>
+          <string>constant.numeric</string>
+        </dict>
+        <key>3</key>
+        <dict>
+          <key>name</key>
+          <string>variable.language.fenced.markdown</string>
+        </dict>
+      </dict>
+      <key>begin</key>
+      <string>\+ (Request|Response) (\d+)? ?\(?(.*)\)?</string>
```

Comment:
I think this regex is incorrect (demonstrated in your screenshot):

![screen shot 2015-04-22 at 15 22 24](https://cloud.githubusercontent.com/assets/44164/7286791/6e7ee79e-e903-11e4-8169-2c5cc38a2b6d.png)

It's highlighting `application/json)` differently to the start bracket for that `(`.